cellprofiler/modules/resizeobjects.py

```diff
@@ -1,6 +1,20 @@
 # coding=utf-8
 
-"""
+import numpy as np
+import cellprofiler.image as cpi
+import cellprofiler.module as cpm
+import cellprofiler.measurement as cpmeas
+import cellprofiler.preferences as cpprefs
+import cellprofiler.setting as cps
+from cellprofiler.setting import YES, NO
+import numpy
+import scipy.ndimage
+import skimage.transform
+import cellprofiler.module
+import cellprofiler.setting
+import _help
+
+__doc__ = """\
```

Comment:
There should be a reference to `HELP_ON_SAVING_OBJECTS` in the "See also" section of this help doc.